velox/expression/VectorFunction.cpp

```diff
@@ -59,6 +60,11 @@ std::shared_ptr<VectorFunction> getVectorFunction(
     });
   }
 
+  if (resultType) {
+    // For Decimal Functions resultType is required to resolve function object.
```

Comment:
nit: update comment to `For certain vector functions that involve decimal types, the return type is required to resolve the function.`

Reply:
Also mention that the return type must be the last argument.